src/Sylius/Behat/Page/Shop/Order/ShowPage.php

```diff
@@ -77,6 +87,7 @@ protected function getDefinedElements()
             'pay_link' => '#sylius-pay-link',
             'payment_method' => '.item:contains("%name%") input',
             'thank_you' => '#sylius-thank-you',
+            'items_text' => 'div.sub.header div.item:nth-child(3)'
```

Comment:
Alphabetical order R.I.P [*] 😢

Reply:
I put it back to life 
![giphy](https://user-images.githubusercontent.com/473193/48232999-dc54db80-e3b3-11e8-843c-90e9d183b7c7.gif)